rclpy_examples/package.xml

```diff
@@ -7,13 +7,15 @@
   <maintainer email="esteve@osrfoundation.org">Esteve Fernandez</maintainer>
   <license>Apache License 2.0</license>
 
+  <exec_depend>example_interfaces</exec_depend>
   <exec_depend>rclpy</exec_depend>
   <exec_depend>std_msgs</exec_depend>
 
   <test_depend>ament_copyright</test_depend>
   <test_depend>ament_pep257</test_depend>
   <test_depend>ament_pep8</test_depend>
   <test_depend>ament_pyflakes</test_depend>
+  <test_depend>example_interfaces</test_depend>
```

Comment:
You shouldn't repeat exec dependencies as test dependencies. Only additional deps should be listed here.